app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardAdvancedOptionsStep/index.js

```diff
@@ -2,15 +2,17 @@ import { connect } from 'react-redux';
 
 import PlanWizardAdvancedOptionsStep from './PlanWizardAdvancedOptionsStep';
 import * as PlanWizardAdvancedOptionsStepActions from './PlanWizardAdvancedOptionsStepActions';
-import { getVMStepSelectedVms } from './PlanWizardAdvancedOptionsStepSelectors';
+import { getVMStepSelectedVms, getVmIdsWithProperty } from './helpers';
 import reducer from './PlanWizardAdvancedOptionsStepReducer';
+import { findEditingPlan } from '../../PlanWizardSelectors';
```

Comment:
Since we use this helper to add `editingPlan` to `mapStateToProps` in multiple containers, might it make sense to do this only once?

Maybe in `PlanWizard/index.js` or `Overview/index.js`.  Then we could just read the `editingPlan` from the redux store in the steps that need it

Reply:
That's a good point. I'll do that.

Reply:
(Note: after discussion with @michaelkro we decided to leave this as-is)